File: CS/MvcDashboard_ServerSideApi/App_Start/DashboardConfig.cs
using DevExpress.DashboardWeb.Mvc;
using System.Web.Routing;

public class DashboardConfig {
    public static void RegisterService(RouteCollection routes) {
        routes.MapDashboardRoute("MarketingDashboard", "MarketingDashboard");
        routes.MapDashboardRoute("SalesDashboard", "SalesDashboard");
    }
}

File: VB/MvcDashboard_ServerSideApi/App_Start/DashboardConfig.vb
Imports DevExpress.DashboardWeb.Mvc
Imports System.Web.Routing

Public Class DashboardConfig
	Public Shared Sub RegisterService(ByVal routes As RouteCollection)
		routes.MapDashboardRoute("MarketingDashboard", "MarketingDashboard")
		routes.MapDashboardRoute("SalesDashboard", "SalesDashboard")
	End Sub
End Class